### PR TITLE
docs: site refactor

### PR DESCRIPTION
**Pull request by kblok**
I created a landing page with some onboarding
image

Unified the readme page with the examples section:
image

### CODE REVIEW

**Comment by kblok** [comment]
take a look at this @arjunattam

**Comment by Meir017** [comment]
@kblok this is probably just nitpicking but
image
I don't think we need to keep the links to the website, we are already in the website ;)

**Comment by kblok** [comment]
No nitpicking @Meir017 :)
Good point. We should consider that if we copy the readme automatically.

**Comment by Meir017** [comment]
I would consider adding icons with useful links 
image
similar to https://webpack.js.org/

**Comment by kblok** [comment]
@Meir017 do you want to create an issue for that?

File: docfx_project/index.md [hunk 1]
```diff
@@ -1,117 +1,67 @@
...
+}
+</style>
+
+<div class="hero">
+  <div class="wrap">
+    <div class="text">
+      <strong>Playwright Sharp</strong>
+    </div>
+    <div class="buttons-unit-small">
+      <a class="version-link" href="https://github.com/microsoft/playwright-sharp/releases">Release Notes</a><span>|</span><a class="github-link" href="https://github.com/microsoft/playwright-sharp">View on Github</a>
```
Comment (mxschmitt) [inline docfx_project/index.md:17]:
```suggestion
      <a class="version-link" href="https://github.com/microsoft/playwright-sharp/releases">Release Notes</a><span>|</span><a class="github-link" href="https://github.com/microsoft/playwright-sharp">View on GitHub</a>
```

I know a bunch of employees from GitHub and we take the right capitalization very seriously.

Comment (kblok) [reply docfx_project/index.md:17]: Too late, PRs accepted 😂

**Pull request merged**